### DISCUSSION TITLE
Deutsche Übersetzung

### DISCUSSION THREAD

**Discussion by Sencaid** [DevToys-app/DevToys#1128] [General]
Ich übersetze zurzeit ins deutsche, aber nicht auf Crowdin. Ich bin auch schon fast fertig. Dieser Theard dient nur zur Information, dass andere nicht auch noch anfangen. Es kann natürlich sein, dass der eine oder andere Fehler vorhanden ist. Das kann man natürlich dann ausbessern.

**Comment by veler** [comment]
Hello,
Please only translate using Crowdin. We won't accept translation directly in the repository.

**Reply by Sencaid** [reply]
I would send you the files, could you please upload them to Crowdin? Otherwise my work would be in vain, and that would be a shame!

**Reply by Sencaid** [reply]
Would that work? Could you please answer me

**Reply by Sencaid** [reply]
@veler i ask you kindly, would you please accept my files and upload them to crowdin. They are completely identical except for the translation. Please 😭
You also stated that you can edit the files yourself in contributing.md

**Reply by veler** [reply]
Right. It's not my preference but it works. Please open a pull request and I will look at it. 🙂

**Reply by Sencaid** [reply]
I will do this when im done with all Files, thanks 👍

**Reply by Sencaid** [reply]
@veler Should I translate Text.resx so that it shows 100% on Crowdin? The Translation is 1:1 equal.

**Reply by veler** [reply]
If there's nothing to translate, then no need. 🙂
